packages/connect/e2e/__fixtures__/getFeatures.ts

```diff
@@ -115,7 +115,6 @@ const tests = [
                 },
             },
             {
-                rules: ['<2.2.1'], // 2.2.1 added PassphraseEntry capability
```

Comment:
shoudln't the fixture be removed here?

Reply:
Yes, https://github.com/trezor/trezor-suite/pull/13222/commits/a1c995b5be9b3ce3399834ca60a593ec97b85327